utils/src/main/java/org/corfudb/utils/lock/LockClient.java

```diff
@@ -129,6 +140,24 @@ public void registerInterest(@NonNull String lockGroup, @NonNull String lockName
         monitorLocks();
     }
 
+    /**
+     * Enforce a transition to NoLeaseState and stop monitoring.
+     **/
+    public void suspendInterest() {
+        log.info("LockClient {} suspendInterest.", clientId);
+        stopMonitorLocks();
+        lock.input(LockEvent.LEASE_REVOKED);
+    }
+
+    /**
+     * Resume the interest inserting an LEASE_REVOIKED event and start monitoring.
+     **/
+    public void resumeInterest() {
```

Comment:
do we need this functionality or is it only for testing?

Reply:
Same as above.